corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -39,7 +42,7 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search Properties" %}</h4>
         </p>
       </div>
     </div>
-    <div class="panel panel-appmanager">
+    <div class="panel panel-appmanager" data-bind="visible: isEnabled">
```

Comment:
This bind should also be removed - right now you can't add a default search filter without first adding a property.

Reply:
```suggestion
    <div class="panel panel-appmanager">
```